### ISSUE TITLE
1 rat = 1bot ?

### ISSUE THREAD

**Issue by Ubop0202** [mvrozanti/RAT-via-Telegram#43]
@mvrozanti hi! once again tnx for this wonderful rat.

for example: i have 2 victims: Do I need to compile for those victims 2 different rats? or I can use 1 rat, with same bot token?

**Comment by Rostam300** [comment]
yes you have compile rat for any new one!! and you have 1 bot for each rat
and you cant creat more than 20 bot on any telegram acc . it is unuseable

**Comment by shuhrat02** [comment]
use 1 bot for different "victims". 
and try comand: 
"/to <COMPUTER_1_NAME>, <COMPUTER_2_NAME> /msg_box Hello HOME-PC and WORK-PC"

**Comment by mvrozanti** [comment]
@shuhrat02 thanks for pointing that out, is this working correctly? I haven't tested it enough I think

**Comment by mvrozanti** [comment]
Thank you for testing these out!

Have you tried increasing the `relax` keyword? It apparently it defaults to 0.1. Maybe each bot should have different values?
